packages/react-cli/src/commands/generateMaterial.ts

```diff
@@ -0,0 +1,47 @@
+import Command from '@oclif/command';
+import { ensureMaterialExists } from '../tasks/ensure-material-exist';
+import { runEmberInteractively } from '../tasks/run-ember';
+import inquirer = require('inquirer');
+
+export class GenerateMaterialCommand extends Command {
+    static description = 'Creates components with Material Components installed';
+
+    static aliases = ['material', 'm'];
+    
+    static examples = [
+        '$ react material component-name',
+        '$ react m path/to/component'
+    ]
+
+    static args = [{ name: 'name', required: true }];
+
+    async run(): Promise<any> {
+        const { args } = this.parse(GenerateMaterialCommand);
+        let options = [];
+
+        await ensureMaterialExists();
+
+        const answers: any = await inquirer.prompt([
+            {
+                type: 'checkbox',
+                message: 'Select Material Components',
+                name: 'materialComponent',
+                choices: [
+                    { name: 'Button', value: 'Button' },
+                    { name: 'TextField', value: 'TextField' },
+                    { name: 'Table', value: 'Table' },
+                    { name: 'Card', value: 'Card' }
+                ]
+            }
+        ])
+        options = [
+            ...answers.materialComponent.map((component: string) => `--${component}`)
+        ]
+
+        let generatorArgs = ['g', 'material', args.name];
+
+        await runEmberInteractively([...generatorArgs, , ...options].join(' '));
+    }
+
+}
+// TODO: handle class and functional components
```

Comment:
what does this mean?

Reply:
Right now when you generate a component, the only option is functional. One of the questions that popped up to me is, "should there be a flag that specifies whether the user can specify which component to build"

Reply:
oh ok, I can answer that: nope.

the new React paradigm is very anti class.

Reply:
👍